apps/meteor/app/2fa/server/code/index.ts

```diff
@@ -92,7 +92,7 @@ export function isAuthorizedForToken(connection: IMethodConnection, user: IUser,
 		return false;
 	}
 
-	if (tokenObject.bypassTwoFactor === true) {
+	if ('bypassTwoFactor' in tokenObject && tokenObject.bypassTwoFactor === true) {
```

Comment:
Is this needed? 👀 the === should fail if bypassTwoFactor is not present

Reply:
Same case of union typecheck